src/storage/storage.cpp

```diff
@@ -182,10 +165,29 @@ bool swapData(Monitor &monitor,
 
     return true;
 }
-}
+} // namespace
 
 Storage::Storage(StorageConfig config_) : config(std::move(config_)) {}
 
+void Storage::readBlocks(const boost::filesystem::path &path,
```

Comment:
Looks like this is a helper function? Is it supposed to be in the anonymous namespace, too, then?

Reply:
Fixed - it's used from outside this class.  I've made it a free function in the `osrm::storage` namespace.